lib/network/modules/components/nodes/shapes/CustomShape.js

```diff
@@ -62,6 +62,11 @@ class CustomShape extends ShapeBase {
       };
     }
 
+    if (drawLater.nodeDimensions) {
+      this.customSizeWidth = drawLater.nodeDimensions.width;
+      this.customSizeHeight = drawLater.nodeDimensions.height;
```

Comment:
Isn't `nodeDimenstions` a function?

Reply:
no, it's actually an object. the docs were wrong. I fixed them